howso/direct/client.py

```diff
@@ -1704,3 +1829,60 @@ def update_session(self, session_id: str, *, metadata: t.Optional[Mapping] = Non
             # of loaded trainees, raise error
             raise HowsoError("Session not found")
         return updated_session
+
+    def get_hierarchy(self, trainee_id: str, *, trainee_path: Collection[str] | None = None) -> dict:
+        """
+        Get the hierarchy schema of a Trainee.
+
+        Parameters
+        ----------
+        trainee_id : str
+            The root Trainee id.
+        trainee_path : Collection of str, optional
+            The path of a sub-Trainee to filter the hierarchy schema to.
+
+        Returns
+        -------
+        dict
+            The hierarchy schema of the Trainee.
+        """
+        if trainee_path is not None:
+            trainee_path = list(trainee_path)
+        return self.execute(trainee_id, "get_hierarchy", {"path_list": trainee_path})
+
+    def combine_trainee_with_subtrainees(
```

Comment:
Does this need a `trainee_path` parameter?  (I'm not 100% clear what the use case would be, admittedly.)